### ISSUE TITLE
[Accessibility]: Form control element <input> has no associated label

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2194]
## Detailed description

> Describe in detail the issue you're having.

This issue is specific to the `<input>` element inside the `<ibm-toggle>` component, which does not have an associated aria-label.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bg

> Is this issue related to a specific component?

`<ibm-table>`

> What did you expect to happen? What happened instead? What would you like to see changed?

Each form control must have an associated aria-label

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 3.3.2 Labels or Instructions WCAG technique H44 WCAG technique ARIA16

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178185963-c8017de2-c9c0-42fa-8266-9a3885b3921f.png)

**Comment by Akshat55** [comment]
Why does it need an aria-label? We pass a label & use a label tag, I don't see why we would need to add an `aria-label`. Also, Accessibility assessment is not point this out for me.
